definitions/divorce/json/FixedLists/FixedLists-general-referral-nonprod.json

```diff
@@ -0,0 +1,114 @@
+[
+  {
+    "LiveFrom": "13/10/2020",
+    "ID": "GeneralReferralReasonEnum",
+    "ListElementCode": "caseworkerReferral",
+    "ListElement": "Caseworker referral",
+    "DisplayOrder": 1
+  },
+  {
+    "LiveFrom": "13/10/2020",
+    "ID": "GeneralReferralReasonEnum",
+    "ListElementCode": "generalApplicationReferral",
+    "ListElement": "General application referral",
+    "DisplayOrder": 2
+  },
+  {
+    "LiveFrom": "13/10/2020",
```

Comment:
`GeneralApplicationFromEnum ` looks a lot like:

```
  {
    "LiveFrom": "08/09/2020",
    "ID": "GeneralEmailPartiesEnum",
    "ListElementCode": "petitioner",
    "ListElement": "Petitioner / Petitioner's Solicitor",
    "DisplayOrder": 1
  },
  {
    "LiveFrom": "08/09/2020",
    "ID": "GeneralEmailPartiesEnum",
    "ListElementCode": "respondent",
    "ListElement": "Respondent / Respondent's Solicitor",
    "DisplayOrder": 2
  },
  {
    "LiveFrom": "08/09/2020",
    "ID": "GeneralEmailPartiesEnum",
    "ListElementCode": "co-respondent",
    "ListElement": "Co-respondent / Co-respondent's Solicitor",
    "DisplayOrder": 3
  },
  {
    "LiveFrom": "08/09/2020",
    "ID": "GeneralEmailPartiesEnum",
    "ListElementCode": "other",
    "ListElement": "Other",
    "DisplayOrder": 4
  }
```

why don't we make `GeneralEmailPartiesEnum` more generic and re-use type?

Reply:
Happy to do so but i remember you did mention that for now we keep them separate, would your preference be to create a common definition?

Reply:
Done